### ISSUE TITLE
Shorten time between results and metrics generation

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/results-analysis#65]
As discussed in https://github.com/Ecosystem-Infra/wpt-results-analysis/issues/63 there are a few different scheduled jobs in the pipeline between results submitted to wpt.fyi and the metrics being generated and used on https://wpt.fyi/interop-2022. We could shorten this time.

**Comment by gsnedders** [comment]
Now we're regenerating all the data from scratch each time, is there any reason not to do it more frequently, and just accept the latest number (for the current day or two) can change over time, as we get more runs?

**Comment by foolip** [comment]
No, I think we should change it to calculate metrics for as recent as there is data.

**Comment by gsnedders** [comment]
With #80 having landed, I think we're at max 4h latency: 1h because the https://github.com/web-platform-tests/results-analysis-cache is updated hourly, then 3h because this is only updated every three hours.

**Comment by gsnedders** [comment]
Reopening because #81 reverted that.
